civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -16,6 +16,10 @@ $row-margin-bottom: 15px;
   margin-bottom: 20px;
 }
 
+.chr_leave-request-modal__footer {
+  margin: 0;
```

Comment:
The margin was off because `margin-left: -1px` was applied to a very large selector from shoreditch, this probably happened because of using `@include` statement. Spent 1 hour but was not able to figure out. Should I spend more time?